controllers/datadoghq/metrics.go

```diff
@@ -29,7 +29,10 @@ const (
 	transitionPromLabel        = "transition"
 	lifecycleStatus            = "lifecycle_status"
 	monitorName                = "monitor_name"
-	monitorNamespace           = "monotor_namespace"
+	monitorNamespace           = "monitor_namespace"
+	clientPromLabel            = "client"
```

Comment:
The `recommender` name was perhaps more meaningful. As it's a not generic HTTP Client but only dedicated to call recommenders.

Reply:
sure, but there metrics could be used for any http call (now that they are in "metrics") - and client contains the url of the recommender when it's a recommender